From: Arnaud Ferraris <aferraris@debian.org>
Date: Mon, 10 Feb 2025 15:27:29 +0100
Subject: Cargo.toml: relax dependencies

Ensure we can use the versions of crates packaged in Debian.

Forwarded: not-needed
---
 Cargo.toml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index b627703..bf90b5d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -8,11 +8,11 @@ glob = "0.3.1"
 greetd_ipc = { version = "0.10.0", features = ["sync-codec"] }
 async-channel = "2.2.1"
 anyhow = "1.0.82"
-libphosh = "0.0.5"
+libphosh = "0.0.6"
 clap = { version = "4.5.4", features = ["derive"] }
 wayland-client = "0.31"
 zbus = { version = "4", default-features = false, features = ["blocking", "async-io"] }
-nix = { version = "0.27", features = ["signal"] }
+nix = { version = "0.29", features = ["signal"] }
 async-global-executor = "2.4.1"
 futures-util = "0.3.30"
 log = "0.4.22"
@@ -35,7 +35,7 @@ glib-build-tools = "0"
 [dev-dependencies]
 input-event-codes = "6.2.0"
 wayland-client = "0.31"
-wayland-protocols = { version = "0.31", features = ["client"] }
+wayland-protocols = { version = "0.32", features = ["client"] }
 wayland-protocols-misc = { version = "0.3.1", features = ["client"] }
-wayland-protocols-wlr = { version = "0.2", features = ["client"] }
+wayland-protocols-wlr = { version = "0.3", features = ["client"] }
 tempfile = "3"
